nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/batch/BatchManagerComponent.java

```diff
@@ -172,6 +177,19 @@ protected Object execute(Object blobInput, String chainOrOperationId,
         }
 
         OperationContext ctx = new OperationContext(session);
+
+        AutomationServer server = Framework.getService(AutomationServer.class);
+        RestBinding binding = server.getOperationBinding(chainOrOperationId);
+
+        if (binding != null && binding.isAdministrator()) {
+            Principal principal = ctx.getPrincipal();
+            if (!(principal instanceof NuxeoPrincipal && ((NuxeoPrincipal) principal).isAdministrator())) {
+                String message = "Not allowed. You must be administrator to use this operation";
```

Comment:
We may want to log the operation / chain id? I'm not sure "this operation" will be useful in the `server.log`.